dali/pipeline/init.cc

```diff
@@ -69,20 +69,10 @@ void DALIInit(const OpSpec &cpu_allocator,
 #if DALI_DEBUG
   subscribe_signals();
 #endif
-  InitializeBackends(cpu_allocator, pinned_cpu_allocator, gpu_allocator);
+  (void)cpu_allocator;
+  (void)pinned_cpu_allocator;
+  (void)gpu_allocator;
```

Comment:
Maybe move this to the beginning of the scope?

Reply:
Done.